### ISSUE TITLE
Package managers section has wrong icons vs description

### ISSUE THREAD

**Issue by juanpicado** [verdaccio/ui#208]
**Describe the bug**

<img width="422" alt="Screen Shot 2019-10-17 at 7 25 27 AM" src="https://user-images.githubusercontent.com/558752/66980029-dda48300-f0af-11e9-9e5f-294020b460e3.png">

**Expected behavior**

- Package managers match with each icon
- Include test that prove this won't happen again

**Comment by V1shvesh** [comment]
Shall I fix this issue?

**Comment by juanpicado** [comment]
Go ahead @V1shvesh :-)

**Comment by V1shvesh** [comment]
Hey @juanpicado! I am trying to run a single test `Install.test.tsx`, but there seems to be an issue while I try to run it.

Command I used to invoke the test: `$ npx jest ./src/components/Install/Install.test.tsx`

Error:
```
● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /home/crysna/Git/verdaccio-ui/src/components/Install/Install.test.tsx:20
    const ComponentToBeRendered = () => <_Version.DetailContext.Provider value={detailContextValue}>
                                        ^

    SyntaxError: Unexpected token <

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:537:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:579:25)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        1.665s
Ran all test suites matching /.\/src\/components\/Install\/Install.test.tsx/i.
```

Am I doing something wrong?

**Comment by juanpicado** [comment]
Use this example as guidance. 
```
 BABEL_ENV=ui yarn jest --config jest/jest.config.js src/components/Versions/Versions.test.tsx --coverage=true -u
```

**Comment by anikethsaha** [comment]
On it 👍

**Comment by anikethsaha** [comment]
you can close this as this has been fixed in #211
